android/composeui/src/main/java/com/stadiamaps/ferrostar/composeui/theme/InstructionRowTheme.kt

```diff
@@ -33,5 +33,5 @@ object DefaultInstructionRowTheme : InstructionRowTheme {
     @Composable get() = MaterialTheme.colorScheme.onSurface
 
   override val backgroundColor: Color
-    @Composable get() = MaterialTheme.colorScheme.surface
+    @Composable get() = MaterialTheme.colorScheme.surfaceContainerLow
```

Comment:
FWIW this came from the Material 3 [spec](https://m3.material.io/components/bottom-sheets/specs)

Reply:
My only comment on this is: 

When material theme is used, the more complex the value is, the less likely someone has it in their theme. This isn't so much a problem. More just a warning when someone asks why their instructions row isn't changing color. It'll be because people rarely set those very nuanced values.